package.json

```diff
@@ -31,6 +35,7 @@
     "emberjs-build": "0.5.3",
     "express": "^4.5.0",
     "finalhandler": "^0.4.0",
+    "fs-extra": "^0.26.3",
```

Comment:
This is used from `blueprints/route/index.js` so it needs to be in `dependencies`.